scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable prefer-arrow-callback */
```

Comment:
This lint rule did not play nice with tests that need the mocha callbacks with `(this)`

Reply:
It looks like this can be removed and the test "should have have triggered cloud functions" can be changed to an arrow function (it specifies `this` as an argument to `function` but doesn't use `this`, so that's why it's failing).